Formula/portable-zlib.rb

```diff
@@ -0,0 +1,31 @@
+require File.expand_path("../../Abstract/portable-formula", __FILE__)
+
+class PortableZlib < PortableFormula
+  desc "Portable zlib"
+  homepage "http://www.zlib.net/"
+  url "http://zlib.net/zlib-1.2.11.tar.gz"
+  mirror "https://downloads.sourceforge.net/project/libpng/zlib/1.2.11/zlib-1.2.11.tar.gz"
+  sha256 "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1"
+
+  # http://zlib.net/zlib_how.html
+  resource "test_artifact" do
+    url "http://zlib.net/zpipe.c"
+    version "20051211"
+    sha256 "68140a82582ede938159630bca0fb13a93b4bf1cb2e85b08943c26242cf8f3a6"
+  end
+
+  def install
```

Comment:
May want to add `ENV.universal_binary if build.with? "universal"` to align with the behaviours of other lib formulae in the repo